File: programmers/Summer-Winter/2018/[Lv1] 소수 만들기.py
from itertools import combinations as comb
def is_prime(num):
    for i in range(3,num,2):
        if num % i==0: return 0
    return 1
def solution(nums):
    answer = 0
    temp=[sum(com) for com in list(comb(nums,3)) if sum(com)%2!=0]
    for t in temp: answer+=is_prime(t)
    return answer

File: programmers/Summer-Winter/2018/[Lv1] 소수 만들기.js
const isPrime = num => {
    for(let i = 2; i <= Math.sqrt(num); i++){
        if(!(num % i)) return 0;
    }
    return 1;
}
function solution(nums) {
    let answer = 0;
    for(let i = 0; i < nums.length; i++){
        for(let j = i + 1; j < nums.length; j++){
            for(let k = j + 1; k < nums.length; k++){
                isPrime(nums[i] + nums[j] + nums[k]) && answer++;
            }
        }
    }
    return answer;
}